src/components/OpenLayersMap.js

```diff
@@ -168,46 +103,89 @@ class OpenLayersMap extends React.Component{
       autoComplete: true,
     });
 
-    const modifyingLayer = new LayerVector({
-      source: modifyingSource,
-      style: getMapStyle
-    });
-
-    const staticLayer = new LayerVector({
-      source: staticSource,
-      style: getMapStyle
+    const resortLayer = new LayerVector({
+      source,
+      style: getMapStyle,
     });
 
     // Orientation
     const projection = Projection.get('EPSG:3857');
     const centerCoords = [-106.553668, 39.612616];
-    
+
     // Map
     const map = new Map({
       loadTilesWhileInteracting: false,
       target: 'map-container',
-      layers: [bingMapsLayer, staticLayer, modifyingLayer],
+      layers: [bingMapsLayer, resortLayer],
       view: new View({
-        projection: projection,
+        projection,
         center: Projection.fromLonLat(centerCoords),
         zoom: 14.2,
-        rotation: 2.4
-      })
+        rotation: 2.4,
```

Comment:
We can leave for now, but rotation should be ommited from the final version since it's a resort specific thing.

Reply:
I'll toss it. 🗑